src/generator/types.ts

```diff
@@ -23,12 +23,22 @@ export interface ParsedField {
   hasDefaultValue?: boolean;
   default?: any;
   apiProperties?: IApiProperty[];
+  createApiResp?: boolean;
+  updateApiResp?: boolean;
+  plainApiResp?: boolean;
   classValidators?: IClassValidator[];
   relationName?: string;
   relationFromFields?: string[];
   relationToFields?: string[];
   pureType?: boolean;
 }
+export function isApiResp(field: ParsedField, dtoType?: string) {
+  if (dtoType) {
+    type ObjectKey = keyof typeof field;
+    const apiResp = (dtoType + 'ApiResp') as ObjectKey;
+    return field[apiResp];
+  }
+}
```

Comment:
This should not be defined in `types.ts`, but in `helpers.ts` or `api-decorator.ts`.